README.adoc

```diff
@@ -0,0 +1,250 @@
+* link:https://github.com/GurbaniNow/gurmukhi-fonts[GurbaniNow/gurmukhi-fonts] are used to display bani and gurbani
+
+## Code of Conduct
+
+Please note that this project is released under the Contributor Covenant. By participating in this project you agree to abide by its terms. Our intention is to signal a safe open-source community by welcoming all people to contribute, and pledging in return to value them as whole human beings and to foster an atmosphere of kindness, cooperation, and understanding.
+
+> We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+> We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+> link:https://www.contributor-covenant.org/version/2/0/code_of_conduct/[The Contributor Covenant]
+
+## License
+
+IMPORTANT: Applies to code and content resting outside of the `data` folder.
+
+The code of the `{repo}` repo is under v3 of the link:LICENSE.adoc[GPL]. It is similar to the Golden Rule: do unto others as you would have them do unto you. In exchange for benefitting from the work completed in this repo, others must share their derivative work under v3 of the link:LICENSE.adoc[GPL].
+
+> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+> This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+> You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+## Gurbani and Panthic Compilations
+
+IMPORTANT: Applies to different texts inside the `data` folder, generated inside the `build` folder, and as releases (e.g. GitHub, npm).
+
+As typical of many old, historical works, most gurbani and panthic texts are free of known copyright restrictions. We identify it as being in the https://creativecommons.org/publicdomain/mark/1.0/[public domain] as a work of factual compilation with originality. The honor and reputation of the original works are to be maintained. Derogatory treatments (including adding to, deleting from, altering of, or adapting) the words in a way that distorts or mutilates the original work is forbidden. That is why, whenever possible, physical sources are used to determine the digital representation of these works as to avoid any controversy.
+
```

Comment:
Need to add:

> Translations, transliterations, notes, compilations, or other items which are not Gurbani and which are created or uniquely organized by the Shabad OS team are subject to the [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/).

Reply:
I don't believe we have any content of our own. If we were to add something of our own, like the translations repo, then this would make sense to me. But it doesn't seem that will happen for quite some time.